samples/highcharts/website/homepage-patterns-demo/demo.js

```diff
@@ -0,0 +1,123 @@
+        })
+    };
+}
+
+// Get 5 patterns
+const patterns = [0, 1, 2, 3, 4].map(getPattern);
+
+const chart = Highcharts.chart('container', {
+    chart: {
+        type: 'pie'
+    },
+
+    title: {
+        text: 'Primary desktop/laptop screen readers',
+        align: 'left'
+    },
+
+    subtitle: {
+        text: 'Source: WebAIM. Click on point to visit official website',
+        align: 'left'
+    },
+
+    colors: patterns,
+
+    tooltip: {
+        valueSuffix: '%',
+        borderColor: '#8ae',
+        shape: 'rect',
+        backgroundColor: 'rgba(255, 255, 255, 0.94)',
+        followPointer: false,
+        stickOnContact: true
+    },
+
+    plotOptions: {
+        series: {
+            dataLabels: {
+                enabled: true,
+                connectorColor: '#777',
+                format: '<b>{point.name}</b>: {point.percentage:.1f} %'
+            },
+            point: {
+                events: {
+                    click: function () {
+                        window.location.href = this.website;
+                    }
+                }
+            },
+            cursor: 'pointer',
+            borderWidth: 3
+        }
+    },
+
+    series: [{
+        name: 'Screen reader usage',
+        data: [{
+            name: 'NVDA',
+            y: 40.6,
+            website: 'https://www.nvaccess.org',
+            accessibility: {
+                description: 'This is the most used desktop screen reader'
+            }
+        }, {
+            name: 'JAWS',
+            y: 40.1,
+            website: 'https://www.freedomscientific.com/Products/Blindness/JAWS'
+        }, {
+            name: 'VoiceOver',
+            y: 12.9,
+            website: 'http://www.apple.com/accessibility/osx/voiceover'
+        }, {
+            name: 'ZoomText',
+            y: 2,
+            website: 'http://www.zoomtext.com/products/zoomtext-magnifierreader'
+        }, {
+            name: 'Other',
+            y: 4.4,
+            website: 'http://www.disabled-world.com/assistivedevices/computer/screen-readers.php'
+        }]
+    }],
+
+    responsive: {
+        rules: [{
+            condition: {
+                maxWidth: 500
+            },
+            chartOptions: {
+                plotOptions: {
+                    series: {
+                        dataLabels: {
+                            format: '<b>{point.name}</b>'
+                        }
+                    }
+                }
+            }
+        }]
+    }
+});
+
+// Toggle patterns enabled
+document.getElementById('patterns').onclick = function () {
+    this.classList.add('active');
+    document.getElementById('default').classList.remove('active');
+    chart.update({
+        colors: patterns
+    });
+};
+
+document.getElementById('default').onclick = function () {
+    this.classList.add('active');
+    document.getElementById('patterns').classList.remove('active');
+    chart.update({
+        colors: pieColors
+    });
```

Comment:
There is a glitch when pressing the Default button, the slices turn black for a fraction of a second, before the solid colors kick in. We can prevent that by disabling animation:
```suggestion
    }, true, true, false);
```

Reply:
I implemented this fix.